protocol/x/clob/e2e/rate_limit_test.go

```diff
@@ -94,6 +95,30 @@ func TestRateLimitingOrders_RateLimitsAreEnforced(t *testing.T) {
 			firstMsg:  &CancelOrder_Alice_Num0_Id0_Clob1_GTB5,
 			secondMsg: &CancelOrder_Alice_Num1_Id0_Clob0_GTB20,
 		},
+		"Batch cancellations with same subaccounts": {
+			blockRateLimitConfig: clobtypes.BlockRateLimitConfiguration{
+				MaxShortTermOrdersAndCancelsPerNBlocks: []clobtypes.MaxPerNBlocksRateLimit{
+					{
+						NumBlocks: 2,
+						Limit:     3,
```

Comment:
for my understanding - why do we need to set limit to 3 here? doesn't batch cancel have a weight of 2? so we can equivalently set this to 2 per 2 blocks?